ios/link-assets-manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "migIndex": 1,
+  "data": [
+    {
+      "path": "assets/fonts/MontserratBold.ttf",
+      "sha1": "4c5b03394078109a583a5ba77c74b045c02e3d9f"
+    },
+    {
+      "path": "assets/fonts/MontserratMedium.ttf",
+      "sha1": "be0644fcab3959ef5dc76188689c31ae7976cda0"
+    },
+    {
+      "path": "assets/fonts/MontserratRegular.ttf",
+      "sha1": "f2ce81e155aed8c2b40a293bfda86a0fd53e24be"
+    },
+    {
+      "path": "assets/fonts/MontserratSemibold-Yz368.ttf",
```

Comment:
what is the difference with android/app/src/main/assets/fonts/MontserratSemibold.ttf ?